client/my-sites/stats/stats-list/stats-list-card.jsx

```diff
@@ -82,18 +83,43 @@ const StatsListCard = ( {
 		);
 	};
 
+	const getLabelIcon = ( labelIconType ) => {
+		if ( labelIconType === 'folder' ) {
+			return <Icon className="stats-icon" icon={ file } size={ 22 } />;
+		} else if ( labelIconType === 'tag' ) {
+			return <Icon className="stats-icon" icon={ tag } size={ 22 } />;
+		}
+	};
+
+	const outputLeftItem = ( item ) => {
+		let leftSideItem; // undefined value avoids rendering an empty node if nothing generates the output
+
+		// left icon visible for avatars, contry flags or tags and categories.
+		if ( item?.countryCode ) {
+			leftSideItem = <StatsListCountryFlag countryCode={ item.countryCode } />;
+		} else if ( showLeftIcon && item?.icon ) {
+			leftSideItem = <StatsCardAvatar url={ item?.icon } altName={ item?.label } />;
+		}
+
+		if ( Array.isArray( item?.label ) ) {
+			// tags without children have one item in its label array;
+			// tags with children have them duplicated in this label array - chevron is added and label is constructed by concatenating items.
+			if ( item?.label?.length === 1 ) {
+				leftSideItem = getLabelIcon( item?.label[ 0 ]?.labelIcon );
```

Comment:
theoretically the first question mark could be saved, because it's already checked in the if condition.